docs/reference/query-dsl/term-query.asciidoc

```diff
@@ -1,168 +1,59 @@
 [[query-dsl-term-query]]
 === Term Query
 
-The `term` query finds documents that contain the *exact* term specified
-in the inverted index.  For instance:
+Returns documents that contain an *exact* term in a provided field.
 
-[source,js]
---------------------------------------------------
-POST _search
-{
-  "query": {
-    "term" : { "user" : "Kimchy" } <1>
-  }
-}
---------------------------------------------------
-// CONSOLE
-<1> Finds documents which contain the exact term `Kimchy` in the inverted index
-    of the `user` field.
-
-A `boost` parameter can be specified to give this `term` query a higher
-relevance score than another query, for instance:
-
-[source,js]
---------------------------------------------------
-GET _search
-{
-  "query": {
-    "bool": {
-      "should": [
-        {
-          "term": {
-            "status": {
-              "value": "urgent",
-              "boost": 2.0 <1>
-            }
-          }
-        },
-        {
-          "term": {
-            "status": "normal" <2>
-          }
-        }
-      ]
-    }
-  }
-}
---------------------------------------------------
-// CONSOLE
-
-<1> The `urgent` query clause has a boost of `2.0`, meaning it is twice as important
-    as the query clause for `normal`.
-<2> The `normal` clause has the default neutral boost of `1.0`.
-
-A `term` query can also match against <<range, range data types>>.
-
-.Why doesn't the `term` query match my document?
```

Comment:
Whilst I'm not wedded to having this explanation in this page of the documentation I think it would be useful to make sure we explain the differece between searching with analyzed or not analyzed queries somewhere since it gives some understanding into how search works and how to avoid some pitfalls. wdyt?

Reply:
Thanks for the feedback.

I think the warning provides enough information for users looking to get started quickly, but it doesn't explain _why_ you should avoid using term-level queries for analyzed fields. The example in the aside is great for that.

I think that content would fit better in a concept-focused page like "Search structured data" or "Search full-text." Those don't exist yet, but I'll work on creating them and add it to this PR.

Reply:
That sounds great, thanks